crates/forge/src/run_tests/helpers.rs

```diff
@@ -0,0 +1,11 @@
+use forge_runner::package_tests::with_config::TestCaseConfig;
+
+#[must_use]
+pub fn skip_fork_tests() -> bool {
+    std::env::var("SNFORGE_SKIP_FORK_TESTS").is_ok_and(|v| v == "1")
```

Comment:
Shouldn't we just check if this variable is set? I don't think value of it matters.